Builds/containers/packaging/dpkg/debian/rules

```diff
@@ -16,31 +16,36 @@ override_dh_systemd_start:
 
 override_dh_auto_configure:
 	env
-	rm -rf bld && mkdir -p bld/rippled
-	cd bld/rippled && \
-	cmake ../.. -G Ninja \
-		-DCMAKE_INSTALL_PREFIX=/opt/ripple \
+	rm -rf bld
+
+	conan export external/snappy snappy/1.1.9@
+
+	conan install . -if bld/rippled  --build missing --settings build_type=Release
```

Comment:
I prefer long options in these scripts, especially if their short form lends itself to being misunderstood.

```suggestion
	conan install . --install-folder bld/rippled  --build missing --settings build_type=Release
```